src/app.py

```diff
@@ -157,9 +196,70 @@ def appium_run(avd_name: str):
             cmd += ' --nodeconfig {file}'.format(file=CONFIG_FILE)
         except ValueError as v_err:
             logger.error(v_err)
+    elif grid4_connect == True:
+        local_ip = os.popen('ifconfig eth0 | grep \'inet\' | cut -d: -f2 | awk \'{ print $2}\'').read().strip()
+        try:
+            mobile_web_test = convert_str_to_bool(str(os.getenv('MOBILE_WEB_TEST', False)))
+            appium_host = os.getenv('APPIUM_HOST', local_ip)
+            appium_port = int(os.getenv('APPIUM_PORT', 4723))
+            plafform_name = 'android'
+            device_name = os.getenv('DEVICE', 'chrome')
+            browser_name = default_web_browser if mobile_web_test else device_name
+            create_node_config_selenium_grid_4(browser_name, appium_host, appium_port, plafform_name)
+            selenium_jar_version = get_selenium_jar_url()
+            logger.info('Latest selenium server jar found: {selenium_jar_version}'.format(selenium_jar_version=selenium_jar_version))
+            download_selenium_server_url = 'https://github.com{selenium_jar_version}'.format(selenium_jar_version=selenium_jar_version)
+            jar_url = '/opt/selenium/'
+            if not os.path.isdir(jar_url):
+                os.mkdir(jar_url)
+            start_selenium_node = "/opt/bin/start-selenium-node.sh"
```

Comment:
No /opt/bin/start-selenium-node.sh file in such location.  File in src folder left

Reply:
Thank you @SHELA, good catch! 
I'll update the path